File: Data Structures & Algorithms/longest-consecutive-sequence/submission-1.java
class Solution {
    public int longestConsecutive(int[] nums) 
    {
        if(nums.length == 0) return 0;
        Arrays.sort(nums);
        HashMap<Integer,Integer> hm = new HashMap<>();
        int ans = Integer.MIN_VALUE;
        for(int i = 0;i<nums.length;i++){
            hm.put(nums[i],hm.getOrDefault(nums[i]-1,0)+1);
            ans = Math.max(hm.get(nums[i]),ans);
        }
        return ans;
    }
}

File: Data Structures & Algorithms/maximum-subarray/submission-1.py
class Solution:
    def maxSubArray(self, nums: List[int]) -> int:
        prefix_sum = 0
        ans = -float('inf')
        for num in nums:
            prefix_sum += num
            ans = max(ans, prefix_sum)
            if prefix_sum < 0:
                prefix_sum = 0
        return ans 
        